lib/maze/api/appium/file_manager.rb

```diff
@@ -0,0 +1,29 @@
+module Maze
+  module Api
+    module Appium
+      # Provides operations for working with files during Appium runs.
+      class FileManager
+        # param driver
+        def initialize
+          @driver = Maze.driver
+        end
+
+        # Creates a file with the given contents on the device (using Appium).  The file will be located in the app's
+        # Documents directory for iOS and /sdcard/Android/data/<app-id>/ for Android.
+        # @param contents [String] Content of the file to be written
+        # @param filename [String] Name (with no path) of the file to be written on the device
+        def write_app_file(contents, filename)
+          path = case Maze::Helper.get_current_platform
+                 when 'ios'
+                   "@#{@driver.app_id}/Documents/#{filename}"
+                 when 'android'
+                   "/sdcard/Android/data/#{@driver.app_id}/#{filename}"
```

Comment:
I'm not sure how robust this will be if we're running tests on a wide range of devices - I think the path to external storage can differ between manufacturers